# Topic of service which will be offered to trigger recognising
# one object
recognize_object_service: "/gazebo_test_tools/recognize_object"

# topic on which object_msgs/Object.msg object information will be published
# after it has been "recognized".
objects_topic: "/gazebo_objects"

# Rate at which to publish collision messages
publish_recognition_rate: 10

# service name on which object_msgs/ObjectInfo.srv object information
# can be requested.
request_object_service: "/gazebo_objects/get_info"

# service of under which a object_msgs_tools/RegisterObject can
# be sent in order to start publishing /tf information about an
# object.
register_object_tf_service: "/object_tf_broadcaster/register_object"
